doc/nrf/ug_wifi.rst

```diff
@@ -0,0 +1,141 @@
+.. _ug_wifi:
+
+Wi-Fi
+#####
+
+.. contents::
+   :local:
+   :depth: 2
+
+Wi-Fi is the branding used by the Wi-Fi Alliance to describe the family of wireless local area network standards specified by the IEEE802.11 Working Group.
+The two industry bodies work closely together, with the Wi-Fi Alliance focusing on the requirements and certification of the technology, and the IEEE focusing on the underlying technical specifications.
+The Wi-Fi Alliance is responsible for the Wi-Fi CERTIFIED accreditation program, which aims to achieve high level of interoperability, reliability, and security compliance.
+
+Wi-Fi is an evolving standard, with new IEEE802.11 standards being incorporated every 4-6 years, along with the corresponding WFA certification program.
+The branding “Wi-Fi 6” is defined by the WFA, and aligns with the IEEE802.11ax specification, but includes all previous versions of the IEEE802.11 specifications dating back to the introduction of Wi-Fi in 1997.
+The evolution from the base standard started with the introduction of IEEE802.11b and IEEE802.11a in 1999, IEEE802.11g in 2003, IEEE802.11n in 2008 (and subsequently branded Wi-Fi 4), IEEE802.11ac in 2014 (subsequently branded Wi-Fi 5), and finally IEEE802.11ax in 2019, branded Wi-Fi 6.
+These standards operate in the unlicensed spectrum of the 2.4 GHz and 5 GHz bands.
+However, 11b and 11g are only applicable in the 2.4 GHz band, while 11a and 11ac are only applicable in the 5 GHz band.
+The IEEE802.11ax standard is also specified for operation in the 6 GHz band, with this variant being branded Wi-Fi 6E to signify extension to the 2.4 and 5 GHz bands used in Wi-Fi 6.
+Wi-Fi 7, based on the IEEE802.11be specification, is due to launch by the end of 2023.
+
```

Comment:
```suggestion
The features supported by nRF70 Series Wi-Fi devices can be found in [link to PSes in infocenter]
```

Reply:
Not available yet.

Reply:
This will be the infocenter landing page for the Product Specs, https://infocenter.nordicsemi.com/topic/ps_nrf7002/keyfeatures_html5.html. This will be available by launch.

Reply:
Added a link.